chanbackup/backup.go

```diff
@@ -18,7 +19,8 @@ type LiveChannelSource interface {
 
 	// FetchChannel attempts to locate a live channel identified by the
 	// passed chanPoint.
-	FetchChannel(chanPoint wire.OutPoint) (*channeldb.OpenChannel, error)
+	FetchChannel(tx kvdb.RTx, chanPoint wire.OutPoint) (
```

Comment:
nit: update Godoc here and in the implementation?

Reply:
updated